roles/migrationcontroller/tasks/main.yml

```diff
@@ -20,14 +23,31 @@
         status:
           phase: Reconciling
 
+  - when: olm_managed and reconcile_migrationtenant
+    block:
+    - k8s:
+        state: "present"
+        definition: "{{ lookup('template', 'migration-tenant.yml.j2') }}"
+
+    - name: Clean conditions and set Reconciling state
+      operator_sdk.util.k8s_status:
+        api_version: migration.openshift.io/v1alpha1
+        kind: MigrationTenant
+        name: "{{ meta.name }}"
+        namespace: "{{ meta.namespace }}"
+        force: true
+        conditions: []
+        status:
+          phase: Reconciling
+
   - set_fact: ui_state="present"
-    when: migration_ui
+    when: migration_ui is defined and migration_ui
```

Comment:
What does this do exactly? At first glace I would have expected:
`when: migration_ui` to be the same as `when: migration_ui is defined and migration_ui` but I'm sure this could be a weird Ansible gotcha.

Reply:
@dymurray This is to address situations where users do not specify these fields in the CR. Currently, we do not default them anywhere. We assume they come through the CR. However, it is not true in case of MigrationTenant. We don't have `migration_velero` variable in it. Also, best to safe guard these vars anyway

Reply:
Got it, thanks!